### PR TITLE
Minutes from coordination meeting 2020-12-03

### PR DESCRIPTION
**Pull request by graeme-a-stewart**

### CODE REVIEW

File: organization/_posts/2020/2020-12-03-coordination.md [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+  2. Bookkeeping processed events
+  3. Intersections of the above and/or analysis experience
+
+So far have some candidate speakers from ATLAS/CMS, Paul will tell us about Belle II global tags (probably in 1st theme). Suggestions for speakers from other communities very welcome.
+
+ATLAS has a new group on databases & metadata -- will try to get the coordinators involved. Does (some fraction of) the Frameworks WG want to come along?
+
+- Yes we agreed this was a very good point to explore, given the different views of metadata from these two communities, but that in the then they are very related.
+
+For later in the year, want to follow up on declarative analysis -- flesh out benchmark challenges into more complete analysis situations. HSF to provide a "conventional" implementation for performance comparison?
```
Comment (amete) [inline organization/_posts/2020/2020-12-03-coordination.md:120]:
```suggestion
For later in the year, we want to follow up on declarative analysis -- flesh out benchmark challenges into more complete analysis situations. HSF to provide a "conventional" implementation for performance comparison?
```

File: organization/_posts/2020/2020-12-03-coordination.md [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+
+- We should make sure that ease of use and clarity are prioritised in the solutions. People developing new tools (IRIS-HEP, for example) are definitely interested in this.
+
+### Detector Simulation
+
+Next HSF Detector Simulation working group topical meeting on Wednesday 16th of December at 4pm (CERN time) devoted to the definition of validation strategies for Deep Learning-based fast simulation models as well as the challenges related to the integration of those tools into detailed-simulation frameworks. (Indico entry to come soon!)
+
+We had a short debriefing of the workshop session and it was very positive to have it as Geant4 co-hosted event. One thing we would like to do is to continue fostering including the Geant4 collaboration in HSF.
```
Comment (amete) [inline organization/_posts/2020/2020-12-03-coordination.md:128]:
```suggestion
We had a short debriefing of the workshop session and it was very positive to have it as a Geant4 co-hosted event. One thing we would like to do is to continue fostering including the Geant4 collaboration in HSF.
```

File: organization/_posts/2020/2020-12-03-coordination.md [hunk 3]
```diff
@@ -0,0 +1,189 @@
...
+
+Talk on `perf` based monitoring 20 Jan. (5pm CERN time):
+
+- Agenda: <https://indico.cern.ch/event/974382/>
+- Title: Profiling Tools: `perf`, `strace`, and more
+- Speaker: Dr Guilherme Amadio (CERN)
+
+### Software Training
+
+- Training hackathon end of this month (16th-18th)!
```
Comment (amete) [inline organization/_posts/2020/2020-12-03-coordination.md:154]:
```suggestion
- Training hackathon at the end of this month (16th-18th)!
```

**Review by amete — APPROVED** [review]
Thanks a lot @graeme-a-stewart. I have just a few minor comments. Otherwise, it looks good to me.

**Comment by graeme-a-stewart** [comment]
Thanks so much for all the corrections @amete - accepted and will squash everything down to accept

**Pull request merged**